<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinearFilter</class>
 <widget class="QDialog" name="LinearFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="showImage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>431</width>
     <height>311</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>431</width>
     <height>311</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="boxFilterPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>149</width>
         <height>141</height>
        </rect>
       </property>
       <attribute name="label">
        <string>方框滤波</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>122</width>
          <height>64</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="boxKernelLabel">
             <property name="text">
              <string>Kernel
Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="boxKernelSizespinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Depth
Default(-1)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="boxKernelDepthspinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="blurFilterPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>149</width>
         <height>141</height>
        </rect>
       </property>
       <attribute name="label">
        <string>均值滤波</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Kernel
Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="blurKernelSizespinBox"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="gaussianBlurPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>149</width>
         <height>141</height>
        </rect>
       </property>
       <attribute name="label">
        <string>高斯滤波</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>131</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Kernel
Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="gaussianKernelSizespinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>SigmaX</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sigmaXspinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>SigmaY</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sigmaYspinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="openImageButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
